www/apps/resources/app/commerce-modules/pricing/tax-inclusive-pricing/page.mdx

```diff
@@ -0,0 +1,74 @@
+export const metadata = {
+  title: `Tax-Inclusive Pricing`,
+}
+
+# {metadata.title}
+
+In this document, you’ll learn about tax-inclusive pricing and how it's used during prices calculation.
+
+## What is Tax-Inclusive Pricing?
+
+A tax-inclusive price is an amount taxes is calculated from rather than added to.
```

Comment:
```suggestion
A tax-inclusive price includes taxes and the tax amount is calculated from the price rather than added to it.
```

Reply:
The sentence repeats "includes" and "taxes" a lot 😅 how about:

```
A tax-inclusive price is a price that includes taxes. The tax amount is calculated from the price rather than added to it.
```

Reply:
Sounds good 😄